### DISCUSSION TITLE
Fritzbox 6670 Cable FW.07.63

### DISCUSSION THREAD

**Discussion by C0nvert** [Freetz-NG/freetz-ng#1026] [Allgemein]
- Fritzbox 6670 Cable 
Not Working on FW 07.63:

- Fritz WebIF Start Page
![image](https://github.com/user-attachments/assets/10c403e4-d37c-4152-b56b-232483c303e4)
![image](https://github.com/user-attachments/assets/77fd08fb-8234-43a6-91d5-a8b6e9c95ad0)

- Working: 
everything else

- Tests
FW 7.90 Beta
Half of Start Page is working, the rest loads infinitely
Tested the same patches on the 6660 , everything was working perfectly

**Comment by fda77** [comment]
thanks four your report. tomorrow i will buy this device to test for you which selection causes your problem

**Reply by C0nvert** [reply]
Did you by chance found anything :)?

**Reply by C0nvert** [reply]
> thanks four your report. tomorrow i will buy this device to test for you which selection causes your problem

Hi, any Updates on this?

**Comment by C0nvert** [comment]
Da ich gemerkt habe, dass die Leute hier alle Deutsch sprechen, schreibe ich mal weiter auf deutsch.

Ich habe 2 weitere Punkte feststellen können:
1. dnsmasq Multid beißen sich 
 Sobald ich dnsmasq an habe , geht bei mir das WLAN nicht.
2. Das Pyload Package: Der Pfad für die rc.pyload Datei, liegt unter diesem Pfad  /mod/etc/init.d/
Villt sollte man das in der Package Beschreibung anpassen.
(2.1 gibt es Pläne für Python 3 und Pyload-ng?)
3. Der RAM verbrauch von der Box bei gleichbleibenden Mods ist um einiges größer, als bei der 6660.
Ich würde heute Abend zuhause mal testen und von beiden Boxen die Screenshots hier reinposten
